hw/ip/aes/doc/_index.md

```diff
@@ -280,6 +304,12 @@ The order in which these registers are written does not matter.
 Anything can be written to the unused key registers.
 For AES-128 and AES-192, the actual initial key used for encryption is formed by using the {{< regref "KEY0" >}} - {{< regref "KEY3" >}} and {{< regref "KEY0" >}} - {{< regref "KEY5" >}}, respectively.
 
+If running in CBC or CTR mode, software must also write the initialization vector (IV) registers {{< regref "IV0" >}} - {{< regref "IV3" >}}.
+These registers are little-endian.
```

Comment:
But CTR mode increment of the IV is big-endian, no?

Reply:
Yes, absolutely. I have put a comment in the doc.